src/realm/parser/driver.hpp

```diff
@@ -197,11 +197,13 @@ class ConstantNode : public ValueNode {
     std::unique_ptr<ConstantMixedList> copy_list_of_args(std::vector<Mixed>&);
     std::unique_ptr<Subexpr> copy_arg(ParserDriver*, DataType, size_t, DataType, std::string&);
     std::unique_ptr<Subexpr> visit(ParserDriver*, DataType) override;
+    Mixed get_value();
     util::Optional<ExpressionComparisonType> m_comp_type;
     std::string target_table;
 
 private:
-    void decode_b64(util::FunctionRef<void(StringData)>);
+    std::string decode_buffer;
```

Comment:
I know we haven't been strict about naming here, but I'd prefer `m_decode_buffer` so that the ownership is clear, especially for a private member.

Reply:
Sure